### PR TITLE
LPS-199103 Adding tests for the option of Type in the Item Action 

### PR DESCRIPTION
**Pull request by JerryAdrianoo**
**Description:**
Adding tests for the option of Type in the Item Action 

**Jira Ticket:**
https://liferay.atlassian.net/browse/LPS-199103

### CODE REVIEW

**Comment by JerryAdrianoo** [comment]
ci:test:frontend

**Comment by JerryAdrianoo** [comment]
ci:test:sf

**Comment by JerryAdrianoo** [comment]
ci:test:frontend

**Comment by JerryAdrianoo** [comment]
ci:stop

**Comment by JerryAdrianoo** [comment]
ci:test:frontend

**Comment by JerryAdrianoo** [comment]
@CarlosBrichete failures are not related :)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 1]
```diff
@@ -108,198 +106,560 @@ definition {
...
 
-	@description = "LPS-191485. Verify if the options GET, POST, PATCH, LPS-199219DELETE are present in the Method dropdown in the Item Action"
-	@priority = 4
-	test AssertAllMethodOptionsArePresentOnItemAction {
-		task ("When clicks on New Action button") {
-			Click(
-				key_text = "New Item Action",
-				locator1 = "Button#ANY_SECONDARY");
+	@description = "LPS-192956 Confirm that When the user selects the Modal option, the following fields are displayed in the modal:Variant,Title,URL,Headless Action Key,Confirmation Message,Message Type"
+	@priority = 5
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:110]: Low this to 3 (Update jira ticket too)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 2]
```diff
@@ -108,198 +106,560 @@ definition {
...
-	test CanAssertThatTheTypeFieldIsDisabledInCreationAction {
-		task ("And creates a new item action") {
-			DataSetAdmin.addActions(
-				actionName = "Edit",
-				actionTab = "Creation Actions",
-				icon = "pencil",
-				type = "Link",
-				url = "https://gist.github.com/deborasoliveira/40716d0b03d33ff7e636ca5d366c79c5#file-link-action-url-example");
+	@description = "LPS-192956 Confirm that all the values set in the creation for all the fields are still present when the Edit modal is opened"
+	@priority = 5
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:249]: Low this to 4 (Update jira ticket too)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 3]
```diff
@@ -108,198 +106,560 @@ definition {
...
+	test AllFieldsArePresentWhenEditASidePanelAction {
+		property portal.acceptance = "true";
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:251]: remove property

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 4]
```diff
@@ -108,198 +106,560 @@ definition {
...
+					locator1 = "Select#GENERIC_SELECT_FIELD",
+					value1 = ${assertFields});
+			}
 		}
 	}
 
-	@description = "LPS-191486. Verify if the user can cancel the deletion of an entry by clicking the Cancel button"
-	@priority = 4
+	@description = "LPS-192956 Confirm that all the values set in the creation for all the fields are still present when the Edit modal is opened"
+	@priority = 5
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:392]: Low this to 4 (Update jira ticket too)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 5]
```diff
@@ -108,198 +106,560 @@ definition {
...
+	test AssertAllFieldsArePresentWhenEditAModalActionOnItemAction {
+		property portal.acceptance = "true";
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:394]: remove property

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 6]
```diff
@@ -108,198 +106,560 @@ definition {
...
+
+			AssertElementPresent(
+				key_fieldName = "add-here-the-title-of-the-modal",
+				locator1 = "DataSet#PRESELECT_VALUE_INPUT",
+				value1 = "Edit item Action");
+		}
+	}
+
+	@description = "LPS-191485. Verify if the options GET, POST, PATCH, LPS-199219DELETE are present in the Method dropdown in the Item Action"
+	@priority = 4
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:472]: Low this to 3 (Update jira ticket too)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 7]
```diff
@@ -108,198 +106,560 @@ definition {
...
+
+			AssertElementNotPresent(
+				key_position = 1,
+				keyName = "Edit",
+				locator1 = "DataSet#FIELDS_TABLE");
+		}
+	}
+
+	@description = "LPS-191486. Verify if the user can cancel the deletion of an entry by clicking the Cancel button"
+	@priority = 4
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:662]: Low this to 3 (Update jira ticket too)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 8]
```diff
@@ -581,6 +941,8 @@ definition {
 	@description = "LPS-191488. Confirm that it's possible to cancel changes in the Actions tab."
 	@priority = 5
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:942]: Low this to 3 (Update jira ticket too)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 9]
```diff
@@ -581,6 +941,8 @@ definition {
...
 	test CanCancelEditions {
+		property portal.acceptance = "true";
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:944]: remove property

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 10]
```diff
@@ -839,7 +1203,9 @@ definition {
 	@description = "LPS-192149. Verify if the user can an Action"
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:1203]: Description is not correct

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 11]
```diff
@@ -888,6 +1254,61 @@ definition {
 		}
 	}
 
+	@description = "LPS-192956 Confirm the Side Panel item action is created and it's present in the item action list"
+	@priority = 5
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:1258]: Low this to 4 (Update jira ticket too)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 12]
```diff
@@ -888,6 +1254,61 @@ definition {
...
+	test CanCreateASidePanelItemAction {
+		property portal.acceptance = "true";
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:1260]: remove property

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 13]
```diff
@@ -888,6 +1254,61 @@ definition {
...
+		task ("And the side panel item action is created") {
+			AssertElementPresent(
+				key_position = 1,
+				keyName = "Edit",
+				locator1 = "DataSet#FIELDS_TABLE");
+		}
+	}
+
 	@description = "LPS-191485. Verify if the use can create a Async item action"
 	@priority = 5
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:1313]: Low this to 4 (Update jira ticket too)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 14]
```diff
@@ -888,6 +1254,61 @@ definition {
...
+
+		task ("And the side panel item action is created") {
+			AssertElementPresent(
+				key_position = 1,
+				keyName = "Edit",
+				locator1 = "DataSet#FIELDS_TABLE");
+		}
+	}
+
 	@description = "LPS-191485. Verify if the use can create a Async item action"
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:1312]: it should say use**r**

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 15]
```diff
@@ -989,6 +1410,171 @@ definition {
 		}
 	}
 
+	@description = "LPS-192956 Confirm that is possible to create a Modal Action with Large variant on the Item Action"
+	@priority = 5
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:1414]: Low this to 4 (Update jira ticket too)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 16]
```diff
@@ -989,6 +1410,171 @@ definition {
...
+	test CanCreateLargeModalItemAction {
+		property portal.acceptance = "true";
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:1416]: remove property

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 17]
```diff
@@ -989,6 +1410,171 @@ definition {
...
+
+			AssertElementPresent(
+				key_position = 1,
+				keyName = "Edit",
+				locator1 = "DataSet#FIELDS_TABLE");
+		}
+	}
+
+	@description = "LPS-192956 Confirm that is possible to create a Modal Actions on the Item Action"
+	@priority = 5
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:1469]: Low this to 4 (Update jira ticket too)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 18]
```diff
@@ -989,6 +1410,171 @@ definition {
...
+	test CanCreateModalItemAction {
+		property portal.acceptance = "true";
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:1471]: remove property

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 19]
```diff
@@ -989,6 +1410,171 @@ definition {
...
+
+			AssertElementPresent(
+				key_position = 1,
+				keyName = "Edit",
+				locator1 = "DataSet#FIELDS_TABLE");
+		}
+	}
+
+	@description = "LPS-192956  Confirm that is possible to create a Modal Action with Small variant on the Item Action"
+	@priority = 5
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:1523]: Low this to 4 (Update jira ticket too)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 20]
```diff
@@ -989,6 +1410,171 @@ definition {
...
+	test CanCreateSmallModalItemAction {
+		property portal.acceptance = "true";
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:1525]: remove property

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 21]
```diff
@@ -1096,64 +1684,311 @@ definition {
...
+
+		task ("Then the list of action items is empty") {
+			AssertElementNotPresent(
+				key_position = 1,
+				keyName = "Headless Item Action",
+				locator1 = "DataSet#FIELDS_TABLE");
+		}
+	}
+
+	@description = "LPS-192956 Confirm that when the user clicks on the Delete option of the Modal item action, the item action is deleted"
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:1699]: This test is not needed. Put this test scenario as Low in Jira ticket and remove it.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 22]
```diff
@@ -989,6 +1410,171 @@ definition {
...
+				value1 = "Your request completed successfully.");
+
+			AssertElementPresent(
+				key_position = 1,
+				keyName = "Edit",
+				locator1 = "DataSet#FIELDS_TABLE");
+		}
+	}
+
 	@description = "LPS-191485. Verify if is possible to delete a Async Item Action created"
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:1578]: This test is not needed. Put this test scenario as Low in Jira ticket and remove it.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 23]
```diff
@@ -1096,64 +1684,311 @@ definition {
...
+		task ("And the side panel item action is deleted") {
+			AssertElementNotPresent(
+				key_position = 1,
+				keyName = "Edit",
+				locator1 = "DataSet#FIELDS_TABLE");
+		}
+	}
+
+	@description = "LPS-192149. Verify if the user can change the entry order with drag and drop in the action entries table"
+	@priority = 4
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:1810]: Low this to 3 (Update jira ticket too)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 24]
```diff
@@ -1096,64 +1684,311 @@ definition {
...
+		task ("Then the item actions are ordered correctly following the changes") {
+			AssertElementPresent(
+				key_position = 2,
+				keyName = "view",
+				locator1 = "DataSet#FIELDS_TABLE");
+		}
+	}
+
+	@description = "LPS-192956 Confirm that user can edit a Modal Item Action"
+	@priority = 5
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:1860]: Low this to 4 (Update jira ticket too)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 25]
```diff
@@ -1096,64 +1684,311 @@ definition {
...
+	test CanEditAModalItemAction {
+		property portal.acceptance = "true";
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:1862]: remove property

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 26]
```diff
@@ -1096,64 +1684,311 @@ definition {
...
+
+			AssertElementPresent(
+				key_position = 1,
+				keyName = "update",
+				locator1 = "DataSet#FIELDS_TABLE");
+		}
+	}
+
+	@description = "LPS-192956 Confirm that when the user modify different fields (variant, label, etc.) the changes are saved"
+	@priority = 5
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:1929]: Low this to 4 (Update jira ticket too)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 27]
```diff
@@ -1478,6 +2313,8 @@ definition {
 	@description = "LPS-191488. Confirm that is not possible to edit the Type field."
 	@priority = 5
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:2314]: Low this to 4 (Update jira ticket too)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 28]
```diff
@@ -1478,6 +2313,8 @@ definition {
...
 	test CannotEditTypeField {
+		property portal.acceptance = "true";
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:2316]: remove property

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 29]
```diff
@@ -1678,6 +2515,130 @@ definition {
 		}
 	}
 
+	@description = "LPS-192956 Confirm that the Title field cannot be empty on the Modal Item Action"
+	@priority = 4
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:2519]: Low this to 3 (Update jira ticket too)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 30]
```diff
@@ -1678,6 +2515,130 @@ definition {
...
+
+		task ("And the changes cannot be saved") {
+			AssertVisible(
+				key_title = "Edit",
+				locator1 = "Header#H2_TITLE");
+		}
+	}
+
+	@description = "LPS-192956 Confirm that the Title field cannot be empty on the Side Panel Item Action"
+	@priority = 4
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:2581]: Low this to 3 (Update jira ticket too)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 31]
```diff
@@ -2085,6 +3048,8 @@ definition {
 	@description = "LPS-191488. Verify if the user can edit a created Action and see the informations"
 	@priority = 5
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:3049]: Low this to 4 (Update jira ticket too)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 32]
```diff
@@ -2085,6 +3048,8 @@ definition {
...
 	test CanSeeModalWhenClicksOnEditAction {
+		property portal.acceptance = "true";
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:3051]: remove property

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 33]
```diff
@@ -2187,6 +3152,8 @@ definition {
 	@description = "LPS-192150. Verify if is possible to see the Action icon in the Data Set fragment."
 	@priority = 5
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:3153]: Low this to 3 (Update jira ticket too)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 34]
```diff
@@ -2187,6 +3152,8 @@ definition {
...
 	test CanViewActionIconInFragment {
+		property portal.acceptance = "true";
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:3155]: remove property

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 35]
```diff
@@ -2534,6 +3501,8 @@ definition {
 	@description = "LPS-192149. Verify if the Label field is mandatory and localized in Action"
 	@priority = 5
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:3502]: Low this to 3 (Update jira ticket too)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 36]
```diff
@@ -2534,6 +3501,8 @@ definition {
...
 	test MandatoryAndLocalizedLabelField {
+		property portal.acceptance = "true";
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:3504]: remove property

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 37]
```diff
@@ -2569,6 +3538,129 @@ definition {
 		}
 	}
 
+	@description = "LPS-192956 Confirm that the mandatory fields cannot be empty after the edition"
+	@priority = 5
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:3542]: Low this to 3 (Update jira ticket too)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 38]
```diff
@@ -2569,6 +3538,129 @@ definition {
...
+	test MandatoryFieldsCannotBeEmptyEditingSidePanelItemAction {
+		property portal.acceptance = "true";
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:3544]: remove property

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 39]
```diff
@@ -2569,6 +3538,129 @@ definition {
...
+
+		task ("And the changes cannot be saved") {
+			AssertVisible(
+				key_title = "Edit",
+				locator1 = "Header#H2_TITLE");
+		}
+	}
+
+	@description = "LPS-192956 Confirm that the mandatory fields cannot be empty after the edition"
+	@priority = 4
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:3603]: Low this to 3 (Update jira ticket too)

**Comment by JerryAdrianoo** [comment]
ci:test:sf

**Comment by JerryAdrianoo** [comment]
@CarlosBrichete done!

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase (hunk 1 above)
Comment (CarlosBrichete) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:110]: Still pending to modify

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase [hunk 40]
```diff
@@ -108,198 +106,560 @@ definition {
...
-	@priority = 4
-	test AssertAllMethodOptionsArePresentOnItemAction {
-		task ("When clicks on New Action button") {
-			Click(
-				key_text = "New Item Action",
-				locator1 = "Button#ANY_SECONDARY");
+	@description = "LPS-192956 Confirm that When the user selects the Modal option, the following fields are displayed in the modal:Variant,Title,URL,Headless Action Key,Confirmation Message,Message Type"
+	@priority = 5
+	test AllFieldsAreInModalTypeOnItemAction {
+		property portal.acceptance = "true";
```
[the remark this answers is not in the document]
Comment (CarlosBrichete) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase:112]: Still pending to modify

**Review by CarlosBrichete — CHANGES_REQUESTED** [review]

**Comment by JerryAdrianoo** [comment]
Okay, now I hope it's finished 🥲

**Review by CarlosBrichete — APPROVED** [review]
LGTM :) But probably you will have to wait for this PR to be merged https://github.com/brianchandotcom/liferay-portal/pull/144707

**Pull request closed without merging**